package.json

```diff
@@ -86,6 +86,7 @@
     "wdio-chromedriver-service": "^5.0.2",
     "webdriverio": "^5.11.1",
     "webpack": "^4.32.2",
+    "webpack-auto-inject-version": "^1.2.2",
```

Comment:
Do we really need this as part of the PR? Ask @sjmiles or try removing?

Reply:
i trying removing it, and all seems to work fine without it.